test/dlc_tests/benchmark/sagemaker/tensorflow/training/test_trcomp_performance.py

```diff
@@ -175,3 +175,75 @@ def test_resnet101_at_fp16(self, instance_type, num_gpus, total_n_gpus, instance
             assert billable>=1000, 'False Positive '+result
             assert billable<=threshold, result
 
+@pytest.mark.flaky(reruns=1)
+@pytest.mark.usefixtures("sagemaker_only")
+class TestNLPModels:
+
+
+    @pytest.mark.model("facebook/bart-base")
+    def test_facebook_bart_base(self, instance_type, num_gpus, total_n_gpus, instance_count, distribution_strategy, caching, tensorflow_training, sagemaker_session, capsys, framework_version):
+        epochs = int(100*total_n_gpus)
```

Comment:
Too many epochs. We don't want to take more than 30 mins for these tests. Please provide an estimate of how long the test takes by running it locally.